### ISSUE TITLE
Add a selectivizr2.min.js

### ISSUE THREAD

**Issue by coliff** [corysimmons/selectivizr2#12]
Would be good to add a minified version for folks to grab.

**Comment by corysimmons** [comment]
True. Do you think you could add an npm precommit script for minifying it?

I think we'd need some sort of testing as well since minification can break things sometimes.

**Comment by coliff** [comment]
I haven't added an npm precommit script before - but I'll take a look into it...

**Comment by corysimmons** [comment]
It's not too hard. Google "npm as a build tool" for a pretty cool article, then "npm precommit" to figure it out. Let me know if you need help. Thanks for your help!
